### ISSUE TITLE
Minor suggestion: Show significant digits

### ISSUE THREAD

**Issue by LSAXYZ321** [Qalculate/qalculate-gtk#559]
In the menu: Mode - decimals
, could there be an added option for choosing shown significant digits instead, as an alternative, basically like "Max digits" ?
For large numbers, it might be too much with many decimals also :-)

BR Lars

**Comment by hanna-kn** [comment]
Why not use Mode➞Precision?

**Comment by LSAXYZ321** [comment]
As I understand: Mode - Precision, controls the amount of digits in internal calculations, if not exact. Am I wrong about that assumption?

**Comment by hanna-kn** [comment]
> As I understand: Mode - Precision, controls the amount of digits in internal calculations, if not exact. Am I wrong about that assumption?

That is correct, although the internal is approximately 30 digits higher (to avoid lower precision than specified in the final results), and if the precision is not enough for the desired number of digits it will be visible in the result (unless Approximation➞Interval Arithmetic has been disabled).

**Comment by LSAXYZ321** [comment]
Thanks for your reply and then it would be nice (if not too difficult) to have the option for choosing shown significant digits as suggested, especially in case that the user prefers a very high internal precision, like > 60 digits etc.
